server/polar/event/endpoints.py

```diff
@@ -88,6 +91,10 @@ async def list(
         False,
         description="When true, filters by parent_id (root events if not specified). When false, returns all events regardless of hierarchy.",
     ),
+    aggregate_costs: bool = Query(
```

Comment:
Not 100% sure about this, or we should make it more generic (aggregate_attribute: MultipleQueryFilter?) to be a bit more forward looking